src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -500,25 +490,23 @@ private final static ScriptInfo loadScript(final @Nullable Config config) {
 								Skript.error("Invalid line in variables section");
 								continue;
 							}
-							String name = ((EntryNode) n).getKey().toLowerCase(Locale.ENGLISH);
+							String key = n.getKey();
+							assert key != null; // Eclipse

```

Comment:
Wait a second... Are you sure that the key cannot be null? I think getKey has been explicitly marked as nullable?

Reply:
getKey is nullable in Node, but non-null in most extending classes, including EntryNode

Reply:
Ah yeah. EntryNode check is somewhere above this diff piece, right? Seems good, then.